keyboards/fleuron/keymaps/default/keymap.c

```diff
@@ -0,0 +1,168 @@
+/* Copyright 2017 REPLACE_WITH_YOUR_NAME
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "fleuron.h"
+
+enum custom_layers {
+  _QWERTY,
+  _LOWER,
+  _RAISE
+};
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  DZ,
+  LOWER,
+  RAISE
+};
+
+/*
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+*/
+
+#define _______ KC_TRNS
+#define XXXXXXX KC_NO
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+[_QWERTY] = KEYMAP(
```

Comment:
This should be changed to `LAYOUT_ortho_6x16` (needs to match the define in the h file for the keyboard).

Reply:
Done :)